.github/template-files/config.yml

```diff
@@ -0,0 +1,50 @@
+conda/governance:
+  # [required] community files
+  - CODE_OF_CONDUCT.md
+
+conda/infrastructure:
+  # [required] general workflows
+  - .github/workflows/cla.yml
+  - .github/workflows/update.yml
+
+  # [optional] to include repo in https://github.com/orgs/conda/projects/2
+  - .github/workflows/issues.yml
+  - .github/workflows/labels.yml
+  - .github/workflows/project.yml
+
+  # [optional] stale bot workflows
+  - .github/workflows/stale.yml
+  - .github/workflows/lock.yml
+
+  # [optional] general processes for the conda org
+  - src: templates/HOW_WE_USE_GITHUB.md
+    dst: HOW_WE_USE_GITHUB.md
+
+  # [optional] standard issue templates
+  - src: templates/issues/bug.yml
+    dst: .github/ISSUE_TEMPLATE/0_bug.yml
+
+  - src: templates/issues/feature.yml
+    dst: .github/ISSUE_TEMPLATE/1_feature.yml
+
+  - src: templates/issues/documentation.yml
+    dst: .github/ISSUE_TEMPLATE/2_documentation.yml
+
+  - src: templates/issues/epic.yml
+    dst: .github/ISSUE_TEMPLATE/epic.yml
+
+  # [optional] standard PR template
+  # - src: templates/pull_requests/news_tests_docs.md
+  #   dst: .github/template-files/templates/pull_request_template_details.md
+  - src: templates/pull_requests/base.md
```

Comment:
@beeankha As a heads-up, this might interfere with #158. I'm going to approve this anyways so we get the other bot goodies rolling out.

Reply:
We might end up commenting out some of the optional sections here. I'd like to see what they do first.

Reply:
https://github.com/conda/conda-package-streaming is an example that syncs these issue templates and doesn't make any further modifications

Reply:
Ok sweet. These do look nice but I would like some of these auto-labeled with some of the labels we created. Is there a way to patch these templates with some customization?

I'm definitely going to disable the `epic` issue though. In our project board, we're using milestones as swim lanes and personally I think that is a little redundant with the `epics` concept.

Reply:
you can see the [templates in conda/infrastructure](https://github.com/conda/infrastructure/tree/main/templates):

some of them include fields that can be templated:

https://github.com/conda/infrastructure/blob/c24a6b6e68f1a4cc50e4cee8d625cf6ffd9861cb/templates/HOW_WE_USE_GITHUB.md?plain=1#L20

```markdown
[workflow-update]: [[ repo.html_url ]]/blob/main/.github/workflows/update.yml
```

others include the option to define additional templates to be injected:

https://github.com/conda/infrastructure/blob/c24a6b6e68f1a4cc50e4cee8d625cf6ffd9861cb/templates/issues/bug.yml#L36

```markdown
[% include 'issue_template_details.yml' ignore missing %]
```

Reply:
currently the templates do not offer additional control over the labels

Reply:
Good to know, thanks for your help on this!